bytestring.cabal

```diff
@@ -108,7 +109,7 @@ common language
 
 library
   import:          language
-  build-depends:   base >= 4.9 && < 5, ghc-prim, deepseq, template-haskell
+  build-depends:   base >= 4.11 && < 5, ghc-prim, deepseq, template-haskell
```

Comment:
How far are we from dropping dependency on `ghc-prim` entirely? I'd be happy to drop more old GHCs if it helps.

Reply:
I think the remaining uses are all in the RealFloat Builder internals. I think it will be easiest to clean those up after #637 lands.